src/pages/workspaces/workspace/Data.js

```diff
@@ -762,6 +765,13 @@ const WorkspaceData = _.flow(
                 wdsSchemaError && h(NoDataPlaceholder, {
                   message: 'WDS is unavailable.'
                 }),
+                !wdsSchemaError && !_.some({ targetWorkspace: { namespace, name } }, asyncImportJobs) && _.isEmpty(wdsSchema) && h(NoDataPlaceholder, {
+                  message: 'No WDS tables have been uploaded.'
+                }),
+                !wdsSchemaError && h(NoDataPlaceholder, {
+                  buttonText: 'Upload TSV',
+                  onAdd: () => setUploadingWDSFile(true)
+                }),
```

Comment:
in lieu of any other UX input, could you match the formatting and behavior for Entity Service?

https://github.com/DataBiosphere/terra-ui/blob/359dfc4fb4a6f5c7e4474e7a0340e70f87685bbc/src/pages/workspaces/workspace/Data.js#L682-L686